addons/event/tests/test_event_internals.py

```diff
@@ -421,6 +421,18 @@ def test_event_seats(self):
         self.assertEqual(event.seats_used, 1)
         self.assertEqual(event.seats_expected, 7)
 
+    def test_event_mail_filter_template_on_event(self):
```

Comment:
Alphabetical ordering of methods is a good plus in tests as it make a match between execution order and file order :)